.github/CONTRIBUTING.md

```diff
@@ -52,7 +53,21 @@ We generate manual pages from source on every release. You do not need to submit
 
 ## Design guidelines
 
-You may reference the [CLI Design System][] when suggesting features, and are welcome to use our [Google Docs Template][] to suggest designs.
+### Proposing a design
+
+You may propose a design to solve an open bug or feature request issue that has both [the `needs-design` and `help-wanted` labels][needs design and help wanted].
+
+To propose a design:
+
+- Open a new issue using the [design proposal issue template](./ISSUE_TEMPLATE/submit-a-design-proposal.md).
+- Include a link to the issue that the design is for.
+- Describe the design you are proposing to resolve the issue, leveraging the [CLI Design System][].
+- Mock up the design you are proposing using our [Google Docs Template][] or code blocks.
```

Comment:
Missing link, here

Reply:
This is intended - it gets rendered to the markdown reference link at the bottom of the page starting at L79. You can omit the content from the second square brackets if the content in the first exactly matches the reference, which is what is happening here. Also note the syntax difference using square brackets instead of square and parenthesis. 

There's not really good docs on reference style links that I could find, but TLDR they're like macros. 

Checkout the rendered page and click on these links to see: https://github.com/cli/cli/blob/efe9841caf9ca58fd9d31441c70a18a8cff022a9/.github/CONTRIBUTING.md